systems/plants/URDFRigidBodyManipulator.cpp

```diff
@@ -711,33 +711,28 @@ bool URDFRigidBodyManipulator::addRobotFromURDFString(const string &xml_string,
 
```

Comment:
i was hoping that, after this commit, we could safely get rid of URDFRigidBodyManipulator?  (I've already zapped it in my branch, which I will pull request after this is merged).